README.rst

```diff
@@ -1,4 +1,4 @@
-|Generic badge|
+Generic badge|
```

Comment:
I looks like you accidentally deleted the first character in this line.

Reply:
Thanks, Sebastian!  You are right about this and the other issue regarding environment variables.